README.md

```diff
@@ -27,255 +20,169 @@ which functions as the data backbone for prominent hedge funds, banks, and finan
+TL;DR:
 
-```bash
-curl https://raw.githubusercontent.com/deephaven/deephaven-core/main/containers/python/SciKit-Learn/docker-compose.yml -O
-docker-compose pull
-docker-compose up -d
+```sh
+# Python
+docker run --rm --name deephaven -p 10000:10000 ghcr.io/deephaven/server:latest
 ```
 
-### Launch: Python with TensorFlow
-
-Run the following commands to launch Deephaven for Python server applications with the [TensorFlow](https://www.tensorflow.org/) module pre-installed.
-
-```bash
-curl https://raw.githubusercontent.com/deephaven/deephaven-core/main/containers/python/TensorFlow/docker-compose.yml -O
-docker-compose pull
-docker-compose up -d
+```sh
+# Groovy
+docker run --rm name deephaven -p 10000:10000 ghcr.io/deephaven/server-slim:latest
 ```
 
-### Launch: Python with example data
-
-Run the following commands to launch Deephaven for Python server applications, with example data.
+### From Python
 
-```bash
-curl https://raw.githubusercontent.com/deephaven/deephaven-core/main/containers/python-examples/base/docker-compose.yml -O
-docker-compose pull
-docker-compose up -d
-```
-
-### Launch: Python with example data and NLTK
+Users who wish to avoid using Docker will likely want to use [pip-installed Deephaven](https://deephaven.io/core/docs/tutorials/quickstart-pip/).
 
-Run the following commands to launch Deephaven for Python server applications, with example data and [NLTK](https://nltk.org/).
+TL;DR
 
-```bash
-curl https://raw.githubusercontent.com/deephaven/deephaven-core/main/containers/python-examples/NLTK/docker-compose.yml -O
-docker-compose pull
-docker-compose up -d
+```sh
+pip install deephaven-server deephaven-ipywidgets
 ```
 
-### Launch: Python with example data and PyTorch
-
-Run the following commands to launch Deephaven for Python server applications, with example data and [PyTorch](https://pytorch.org/).
+Then, from Python:
 
-```bash
-curl https://raw.githubusercontent.com/deephaven/deephaven-core/main/containers/python-examples/PyTorch/docker-compose.yml -O
-docker-compose pull
-docker-compose up -d
+```python
+from deephaven_server import Server
+s = Server(port=10000, jvm_args=["-Xmx4g"]).start()
 ```
 
-### Launch: Python with example data and SciKit-Learn
-
-Run the following commands to launch Deephaven for Python server applications, with example data and [SciKit-Learn](https://scikit-learn.org/stable/).
-
-```bash
-curl https://raw.githubusercontent.com/deephaven/deephaven-core/main/containers/python-examples/SciKit-Learn/docker-compose.yml -O
-docker-compose pull
-docker-compose up -d
-```
+### Built from source
 
-### Launch: Python with example data and TensorFlow
+Users who wish to modify source code and contribute to the project should build Deephaven from source. For full instructions, see [how to build Deephaven from source](https://deephaven.io/core/docs/how-to-guides/launch-build/).
```

Comment:
I assume "how to build Deephaven from source" is a title and should probably be capitalized accordingly.